cluster_agent/settings.py

```diff
@@ -26,11 +26,11 @@ class Settings(BaseSettings):
     SENTRY_DSN: Optional[AnyHttpUrl] = None
     SENTRY_ENV: str = "local"
 
-    # Auth0 config for machine-to-machine security
-    AUTH0_DOMAIN: str = "omnivector.us.auth0.com"
-    AUTH0_AUDIENCE: str = "https://armada.omnivector.solutions"
-    AUTH0_CLIENT_ID: str
-    AUTH0_CLIENT_SECRET: str
+    # OIDC config for machine-to-machine security
+    OIDC_DOMAIN: str
+    OIDC_AUDIENCE: str = "https://apis.omnivector.solutions"
+    OIDC_CLIENT_ID: str
```

Comment:
Do we need client secret here? Just to double check, because we dropped it on Jobbergate

Reply:
yes, we need. The main difference is conceptual:

Both applications are machine-to-machine applications but the Jobbergate CLI authenticates against the device code authorization while the cluster agent authenticates against the token authorization

Reply:
I see, thanks